rwengine/src/dynamics/CollisionInstance.hpp

```diff
@@ -25,8 +25,8 @@ class CollisionInstance {
     ~CollisionInstance();
 
     bool createPhysicsBody(GameObject* object, CollisionModel* collision,
-                           DynamicObjectData* dynamics = nullptr,
-                           VehicleHandlingInfo* handling = nullptr);
+        DynamicObjectData* dynamics = nullptr,
```

Comment:
Meanwhile asterisks here is in the right place. What's up with inconsistent formatting?

Reply:
I had PointerAlignment set to the wrong type.